superset/assets/src/messageToasts/stylesheets/toast.less

```diff
@@ -24,7 +24,7 @@
   left: 50%;
   transform: translate(-50%, 0);
   width: 600px;
-  z-index: 3000; // top of the world
+  z-index: @z-max; // top of the world
```

Comment:
comment probably isn't required now right?

Reply:
quite right! Thought I nabbed that.